MovieTracker/Scenes/TVSeriesOnAirView/TVSeriesOnAirDetailsView/TVSeriesOnAirDetailsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TVSeriesOnAirDetailsViewModel.swift
+//  MovieTracker
+//
+//  Created by Andria Inasaridze on 06.01.24.
+//
+
+import SwiftUI
+import NetworkManager
+
+final class TVSeriesOnAirDetailsViewModel: ObservableObject {
+    
+    // MARK: - Properties
+    let id: Int
+    let networkManager: APIServices
+    @Published var posterPath = ""
+    @Published var name = ""
+    @Published var tvSeriesOnAirImage = Image(systemName: "photo")
+    @Published var overview = ""
+    @Published var numberOfEpisodes = 0
+    @Published var numberOfSeasons = 0
```

Comment:
მეც მსგავსად მქონდა ყველა ფროფერთი ცალ-ცალკე შექმნილი. ჰოდა სოფომ რევიუში მიმიკომენტარა, რო ცალკე სტრაქტის შექმნა შეიძლება, რომელსაც ყველა ეს ფროფერთი ექნება. და აქ ესე ჩამოწერა აღარ მოგიწევს.

Reply:
კარგი სოლუშენია ამას სამომავლოდ გავითვალისწინებ, ეხლა რადგან ძაან ბევრი ცვლადიც არ მაქვს დავტოვებ ბარემ, მარა i like it